addons/settings/fnc_init.sqf

```diff
@@ -166,7 +166,7 @@ GVAR(default) setVariable [_setting, [_defaultValue, _setting, _settingType, _se
 private _settingInfo = GVAR(userconfig) getVariable _setting;
 
 if (isNil "_settingInfo") then {
-    private _settingsHash = profileNamespace getVariable [QGVAR(hash), HASH_NULL];
+    private _settingsHash = GVAR(serverHashNamespace) getVariable [QGVAR(hash), HASH_NULL];
```

Comment:
Not sure, but won't this cause clients to use the wrong namespace?

Reply:
We might enable the feature only on servers as you've suggested below: https://github.com/CBATeam/CBA_A3/pull/1617#discussion_r1360016023